### PR TITLE
Fix tools drawer on mobile

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

As title. 

**Other code changes:**
- Makes `truncateEthAddress` util function to use in `FormattedAddress` and `EtherscanLink`

## Screenshots or screen recordings

BEFORE:

https://user-images.githubusercontent.com/96150256/185392333-6c0d15ee-38a7-48cc-abb4-0d1382708930.mp4

AFTER

https://user-images.githubusercontent.com/96150256/185392359-7f9b18f2-c3a2-4a11-a0b3-02af94a74234.mp4

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by johnnyd-eth** [comment]
Mind taking a look at this when you get a chance @peripheralist ?

File: src/components/v2/V2Project/ProjectPayersModal.tsx [hunk 1]
```diff
@@ -69,7 +72,12 @@ export default function ProjectPayersModal({
                   }}
                 >
                   <span style={{ textDecoration: 'underline' }}>
-                    <EtherscanLink value={p.address} type={'address'} />
+                    <EtherscanLink
+                      value={p.address}
+                      type={'address'}
+                      truncated={isMobile ? true : false}
+                      truncateTo={isMobile ? 8 : undefined}
```
Comment (tomquirk) [inline src/components/v2/V2Project/ProjectPayersModal.tsx:79]: **question** why do we need both props? Isn't `truncateTo` sufficient enough?

File: src/utils/formatAddress.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// Truncates an ETH address to specific number of characters (truncateTo)
+// e.g. truncateEthAddress({
+//   address: '0xf0FE43a75Ff248FD2E75D33fa1ebde71c6d1abAd,
+//   truncateTo: 4
+// }) = '0xf0FE...abAd'
+export function truncateEthAddress({
```
Comment (tomquirk) [inline src/utils/formatAddress.ts:6]: **suggestion (non-blocking)** want to have a crack at writing a test?

Comment (tomquirk) [reply src/utils/formatAddress.ts:6]: meh, cbf. dw

**Comment by tomquirk** [comment]
@johnnyd-eth maybe we can fix the whack button in a follow-up.


Screen Shot 2022-08-19 at 9 59 58 AM

**Review by tomquirk — APPROVED** [review]

**Comment by johnnyd-eth** [comment]
> @johnnyd-eth maybe we can fix the whack button in a follow-up.
> 
> Screen Shot 2022-08-19 at 9 59 58 AM

Thanks @tomquirk I just fixed it! Will merge

**Pull request merged**